code/wdb/datastripe.q

```diff
@@ -0,0 +1,66 @@
+\d .ds
+
+segmentid: "J"$.proc.params[`segid]		// segmentid variable defined by applying key to dictionary of input values
+
+td:hsym `$getenv`KDBTAIL
+
+\d .
+
+.wdb.datastripeendofperiod:{[currp;nextp;data]
+    .lg.o[`reload;"reload command has been called remotely"];
+    
+    // remove periods of data from tables
+    t:tables[`.] except .wdb.ignorelist;
+    lasttime:currp-.ds.periodstokeep*(nextp-currp);
+    tabs:{![x;enlist (<;y;z);0b;0#`]}'[t;`time;lasttime];
```

Comment:
Is this the same as the lambda I flagged in common? If so looks like it should be spearated.